index.html

```diff
@@ -0,0 +1,134 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+  <div id="page-wrapper">
```

Comment:
Good Job so far 👏
Please, I kindly suggest using a semantic tag here instead of a `<div>` tag. 

Please as you can see your `<body>` tag performs the same task as your `<div>` tag. Please consider removing the extra `<div id="page-wrapper">` so as to remove the extra node 🤓 🛠️

example:

```
<body>
    <header id="header">
        <!-- Toolbar code goes here -->
    </header>

    <section id="hero"> <!-- replace your <div> tag -->
        <!-- Headline section goes here -->
    </section>
</body>
```